corehq/util/timer.py

```diff
@@ -201,12 +201,6 @@ def to_list(self, exclude_root=False):
         """Get the list of ``NestableTimer`` objects in hierarchy order"""
         return self.root.to_list(exclude_root)
 
-    def __repr__(self):
-        return "TimingContext(root='{}')".format(
-            self.root,
-            self.root.to_dict()
```

Comment:
Hmm, I was only suggesting to remove this one line (`self.root.to_dict()`), not the entire `__repr__` function, which I think is useful to keep around for debugging. Sorry for the not-very-clear https://github.com/dimagi/commcare-hq/pull/25106#pullrequestreview-276676713.

Reply:
Gotcha! That makes a lot of sense